small-talk/db/registerToDB.js

```diff
@@ -1,14 +1,17 @@
 "use server"
 const uri = "mongodb+srv://smt_root:pokemonwithguns@smalltalkcluster0.jo4jne6.mongodb.net/?retryWrites=true&w=majority"
 import Mongoboi from "./mongo"
+import bcrypt from "bcryptjs";
 export default async function handleRegister(username, password, firstname, lastname, dob){ // function that sends the credentials to the database
   const mongoboi = new Mongoboi(uri, "Users");
   const date = new Date(dob).getTime() / 1000; // return the inputted date as a Unix timestamp
   await mongoboi.connect();
 
+  const hashedpass = await bcrypt.hash(password, 10);
```

Comment:
Great job salting. I would suggest to also find a way to generate a new salt for each user instance.

Reply:
Thank you. We will look into how to implement this properly. Good idea